### PR TITLE
Add optional arguments for sendtoaddress

### PR DESCRIPTION
**Pull request by abrkn**
I need to specify replacable. The code is a little awkard, not sure how to improve the style.

### CODE REVIEW

**Comment by MrHash** [comment]
Ok had a quick look but want to go a bit deeper so give me a few hours.

File: src/BitcoinJsonRpc.ts [hunk 1]
```diff
@@ -98,12 +98,36 @@ export default class BitcoinJsonRpc {
...
+    const params: any[] = [address, amount];
+
+    // Argument #6
+    if (replaceable !== undefined) {
+      params.push(comment ?? '', commentTo ?? '', subtractFeeFromAmount ?? false, replaceable);
+    }
+
+    // Argument #5
+    if (subtractFeeFromAmount !== undefined) {
+      params.push(comment ?? '', commentTo ?? '', subtractFeeFromAmount);
```
Comment (MrHash) [inline src/BitcoinJsonRpc.ts:117]: Looks like there's potential to push too many params into the array if multiple arguments are defined? Why not just push param into array in the order seen?

`if (comment !== undefined) { params.push(comment); }` etc.

Comment (abrkn) [reply src/BitcoinJsonRpc.ts:115]: You are right, @MrHash!

**Comment by abrkn** [comment]
When the PR is ready, use `squash and merge` to keep the commit log clean.

**Pull request merged**